org.freecad.FreeCAD.yaml

```diff
@@ -0,0 +1,801 @@
+app-id: org.freecad.FreeCAD
+default-branch: beta
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: FreeCAD
+desktop-file-name-prefix: "(beta) "
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```

Comment:
does this work better than with Qt 5.15? I remember with wayland it was very buggy.

Reply:
better than qt5 but still has some problems, we need people using wayland and reporting the problems though